orm_lib/src/SqlBinder.cc

```diff
@@ -266,22 +259,59 @@ SqlBinder &SqlBinder::operator<<(std::nullptr_t nullp)
     return *this;
 }
 
+SqlBinder &SqlBinder::operator<<(DefaultValue dv)
+{
+    (void)dv;
+    if (type_ == ClientType::PostgreSQL)
+    {
+        utils::replaceAll(*sqlPtr_,
+                          "$" + std::to_string(parametersNumber_ + 1),
+                          "default");
+
+        // decrement all other $n parameters by 1
+        int i = parametersNumber_ + 2;
```

Comment:
`parametersNumber_` is `size_t`. Please also use `size_t`. (So there's no way the code underflow)